crates/adapters/src/transport/kafka/ft/mod.rs

```diff
@@ -115,6 +116,10 @@ fn kafka_config(
     // standard location (e.g., /etc/ssl/).
     set_option_if_missing(&mut settings, "ssl.ca.location", "probe");
 
+    // Enable client context `stats` callback so we can periodically check
+    // up on librdkafka memory usage.
+    set_option_if_missing(&mut settings, "statistics.interval.ms", "1000");
```

Comment:
Not sure how expensive it is for rdkafka to collect these stats, but it's a pretty big object. I wonder if 10 seconds would be a safer polling interval.

Reply:
I'll change it.